ownCloud/Release Notes/ReleaseNotesHostViewController.swift

```diff
@@ -0,0 +1,211 @@
+//
+//  ReleaseNotesHostViewController.swift
+//  ownCloud
+//
+//  Created by Matthias Hühne on 04.12.19.
+//  Copyright © 2019 ownCloud GmbH. All rights reserved.
+//
+
+/*
+ * Copyright (C) 2019, ownCloud GmbH.
+ *
+ * This code is covered by the GNU Public License Version 3.
+ *
+ * For distribution utilizing Apple mechanisms please see https://owncloud.org/contribute/iOS-license-exception/
+ * You should have received a copy of this license along with this program. If not, see <http://www.gnu.org/licenses/gpl-3.0.en.html>.
+ *
+ */
+
+import UIKit
+import ownCloudSDK
+
+class ReleaseNotesHostViewController: UIViewController {
+
+	// MARK: - Constants
+	private let cornerRadius : CGFloat = 8.0
+	private let padding : CGFloat = 20.0
+	private let smallPadding : CGFloat = 10.0
+	private let buttonHeight : CGFloat = 44.0
+	private let headerHeight : CGFloat = 60.0
+
+	// MARK: - Instance Variables
+	var titleLabel = UILabel()
+	var proceedButton = UIButton(type: .roundedRect)
+	var footerLabel = UILabel()
+
+	override func viewDidLoad() {
+		super.viewDidLoad()
+
+		Theme.shared.register(client: self)
+
+		VendorServices.setUserPreferenceValue(NSString(utf8String: VendorServices.shared.appVersion), forClassSettingsKey: .lastSeenReleaseNotesVersion)
+
+		let headerView = UIView()
+		headerView.backgroundColor = .clear
+		headerView.translatesAutoresizingMaskIntoConstraints = false
+		view.addSubview(headerView)
+		NSLayoutConstraint.activate([
+			headerView.leadingAnchor.constraint(equalTo: view.safeAreaLayoutGuide.leadingAnchor),
+			headerView.trailingAnchor.constraint(equalTo: view.safeAreaLayoutGuide.trailingAnchor),
+			headerView.topAnchor.constraint(equalTo: view.safeAreaLayoutGuide.topAnchor),
+			headerView.heightAnchor.constraint(equalToConstant: headerHeight)
+		])
+
+		titleLabel.translatesAutoresizingMaskIntoConstraints = false
+		titleLabel.setContentHuggingPriority(UILayoutPriority.defaultLow, for: NSLayoutConstraint.Axis.horizontal)
+
+		titleLabel.text = "New in ownCloud".localized
+		titleLabel.textAlignment = .center
+		titleLabel.numberOfLines = 0
+		titleLabel.font = UIFont.systemFont(ofSize: UIFont.systemFontSize * 1.5, weight: .bold)
```

Comment:
@hosy can we use dynamic type font for this? Something like `.headline` style or similar.

Reply:
fixed

Reply:
fixed